src/catalog/postgres/pg_type_impl.cpp

```diff
@@ -111,7 +111,8 @@ void PgTypeImpl::BootstrapTypes(const common::ManagedPointer<DatabaseCatalog> db
   insert_base_type(type::TypeId::SMALLINT, "smallint", sizeof(int16_t));
   insert_base_type(type::TypeId::INTEGER, "integer", sizeof(int32_t));
   insert_base_type(type::TypeId::BIGINT, "bigint", sizeof(int64_t));
-  insert_base_type(type::TypeId::DECIMAL, "decimal", sizeof(double));
+  insert_base_type(type::TypeId::REAL, "real", sizeof(double));
+  insert_base_type(type::TypeId::DECIMAL, "decimal", sizeof(int128_t));
```

Comment:
just to confirm is this the same as 

InsertType(txn, type::TypeId::FIXEDDECIMAL, "fixeddecimal", postgres::NAMESPACE_CATALOG_NAMESPACE_OID, sizeof(int128_t), true,
             postgres::Type::BASE);

Reply:
I believe so, yeah.